docs/self-managed/platform-deployment/manual.md

```diff
@@ -168,18 +168,58 @@ To update Tasklist versions, visit the [guide to update Tasklist](../../componen
 
 ## Run Connectors
 
-The [Connector runtime environment](https://repo1.maven.org/maven2/io/camunda/spring-zeebe-connector-runtime) picks up outbound Connectors available on the `classpath` automatically.
-It uses the default configuration specified by a Connector through its `@OutboundConnector` annotation.
+### Bundle
 
-To run the [REST Connector](https://search.maven.org/artifact/io.camunda.connector/connector-http-json) with the runtime environment, execute the following command:
+Bundle includes runtime with all Camunda official connectors.
```

Comment:
Do you just mean Camunda's available Connectors here?

Reply:
That's right, all Camunda's available Connectors

Reply:
Adjusted in latest commit 👍